.gitignore

```diff
@@ -1,3 +1,5 @@
 rethinkdb_data/
+src/homegpg/
+src/__pycache__
```

Comment:
`__pycache__/` rule below already covers `src/__pycache__`, no need to add it separately.